test/multiverse/suites/rails/Envfile

```diff
@@ -7,12 +7,10 @@ RAILS_VERSIONS = [
   ['7.0.4', 2.7],
   ['6.1.7', 2.5],
   ['6.0.6', 2.5, 2.7],
-  ['5.2.8', 2.2, 2.7],
-  ['5.1.7', 2.2, 2.7],
-  ['5.0.7', 2.2, 2.7],
-  ['4.2.11', 2.2, 2.3],
-  ['4.1.16', 2.2, 2.3],
```

Comment:
The CI failed testing with Rails 4.0 / 4.1 and Ruby 2.4. A low percentage of customers are on ruby agent 8.x and Rails 4.x.